common/types.go

```diff
@@ -357,3 +282,314 @@ func (e *ExecutionPayload) TxNum() int {
 	}
 	return len(e.Bellatrix.Transactions)
 }
+
+type BuilderSubmitBlockRequest struct {
```

Comment:
Notice that you are using this type in two ways: either through the methods, which return different things based on whether it's bellatrix or capella, and then there are functions that are specific to bellatrix or capella.

I'd say all of the functions should be specific to bellatrix or capella.